### PR TITLE
11.0 imp account financial reporting

### PR DESCRIPTION
**Pull request by mpanarin**
Allows filtering by account journal in general ledger and trial balance

### CODE REVIEW

**Comment by fclementic2c** [comment]
@mpanarin I will not be able to test before my holidays return sorry (08/08)

**Comment by fclementic2c** [comment]
perfect
thanks

**Review by fclementic2c — APPROVED** [review]

File: account_financial_report/wizard/general_ledger_wizard_view.xml [hunk 1]
```diff
@@ -30,13 +30,15 @@
...
+                    <label for="account_journal_ids"/>
+                    <field name="account_journal_ids" nolabel="1" options="{'no_create': True}"/>
                     <group/>
                     <label for="account_ids"/>
                     <group col="4">
                         <field name="receivable_accounts_only"/>
                         <field name="payable_accounts_only"/>
                     </group>
-                    <field name="account_ids" widget="many2many_tags"  nolabel="1" options="{'no_create': True}"/>
+                    <field name="account_ids" nolabel="1" options="{'no_create': True}"/>
```
Comment (pedrobaeza) [inline account_financial_report/wizard/general_ledger_wizard_view.xml]: Why do you remove this? It's outside the scope of the PR.

File: account_financial_report/wizard/trial_balance_wizard_view.xml [hunk 2]
```diff
@@ -28,13 +28,15 @@
...
+                    <label for="journal_ids"/>
+                    <field name="journal_ids" nolabel="1" options="{'no_create': True}"/>
                     <group attrs="{'invisible':[('show_partner_details','!=',True)]}"/>
                     <label for="account_ids"/>
                     <group col="4">
                         <field name="receivable_accounts_only"/>
                         <field name="payable_accounts_only"/>
                     </group>
-                    <field name="account_ids" widget="many2many_tags" nolabel="1" options="{'no_create': True}"/>
+                    <field name="account_ids" nolabel="1" options="{'no_create': True}"/>
```
Comment (pedrobaeza) [inline account_financial_report/wizard/trial_balance_wizard_view.xml]: And this

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by fclementic2c** [comment]
@pedrobaeza It was my request (but not clearly described here)

This widget change will allow to select several accounts more easily. With the tag widget it is a real pain.
 same comment for : https://github.com/OCA/account-financial-reporting/pull/445

**Comment by pedrobaeza** [comment]
I prefer not to do it. It takes a lot of space in screen. If you want to select more easily, just install https://github.com/OCA/web/tree/11.0/web_widget_many2many_tags_multi_selection

**Review by yvaucher — APPROVED** [review]
LGTM

Please resolve the issue raised by @pedrobaeza

**Comment by fclementic2c** [comment]
@pedrobaeza ok it is a possibility.

**Comment by mpanarin** [comment]
@pedrobaeza updated

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
As a side note, for v12, we should really rid off all these SQL expressions in favor of read_group calls, that they have the same performance, but far better code maintainability and readiness.

**Pull request merged**